packages/opal-client/opal_client/data/updater.py

```diff
@@ -327,109 +330,38 @@ async def update_policy_data(
                 if entry.topics
                 and not set(entry.topics).isdisjoint(set(self._data_topics))
             ]
-            urls = [(entry.url, entry.config, entry.data) for entry in entries]
-
-        if len(entries) > 0:
-            logger.info("Fetching policy data", urls=repr(urls))
+            urls = [(entry.url, entry.config, entry.data)
+                    for entry in entries if entry.save_method.upper() != "DELETE"]
```

Comment:
I don't really like giving "DELETE" a special care here.
url can be None and handler would just not fetch anything, So I feel like being explicit here overall makes the code less readable.